plane_app/scraper.py

```diff
@@ -0,0 +1,65 @@
+from selenium import webdriver
+import datetime
+class Airport:
+
+    def __init__(self):
+        options = webdriver.ChromeOptions()
+        options.add_argument('headless')
+
+        self.url = None
+        self.browser = webdriver.Chrome(executable_path='/home/dawid/selenium/chromedriver',options= options)
+        self.tags = ['tr', 'td']
+        self.head = []
+
+    @property
+    def get_data(self):
+        self.browser.get(self.url)
+        elements = self.browser.find_elements_by_tag_name(self.tags[0])
+        for element in elements:
+            element = element.find_elements_by_tag_name(self.tags[1])
+            element = map(lambda x: x.text, element)
+            element = [x for x in element if x  ]
+            yield (dict(zip(self.head, element)))
+
+
+    @property
+    def save_to_txt(self):
+        file = open(f'{self._class_name()}-{datetime.datetime.now().strftime("%d:%m:%Y %H:%M:%S")}.','w')
+        for data in self.get_data:
+            if data != {}:
```

Comment:
get_data zdaje się zawsze zwraca słownik (yield dict(...))
można to uprościć do:
`if not data:` # data jest Fałszem, czyli: False, '', {}, [], None